CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
   request to the web service.
 * `MaxMind::GeoIP2::Client` now includes the version of Ruby, the version
   of the HTTP client library, and its own version in the User-Agent header.
+* The `anycast?` method was added to `MaxMind::GeoIP2::Record::Traits`.
+  This returns `true` if the IP address belongs to an [anycast
+  network](https://en.wikipedia.org/wiki/Anycast anycast network). This is
```

Comment:
GitHub isn't rendering this correctly. It must have gotten pickier about newlines recently or something.

Reply:
Actually, the issue is the " anycast network" after the link.

Reply:
Ah thanks. Oops. I can fix it.